aws/cloud-formation-template.yml

```diff
@@ -1013,6 +1015,12 @@ Resources:
                 else HOST=$LOC_IPV4
                 fi
                 sed -i -e "s~INJECTHOST~$HOST~" /etc/sysconfig/airflow
+                
+                if [ ! -f /mnt/efs/fernet_key.txt ]; then
+                  FERNET_KEY=$(python -c "from cryptography.fernet import Fernet; FERNET_KEY = Fernet.generate_key().decode(); print(FERNET_KEY)")
+                  echo $FERNET_KET > /mnt/efs/fernet_key.txt
+                  sed -i -e "s~INJECT_FERNET_KEY~$FERNET_KEY~" /etc/sysconfig/airflow
```

Comment:
Still not quite there yet, because the second time the stack is deployed, the key won't be re-generated but it won't be injected either. I think it's best to put this in a separate command with a test directive.

Reply:
It's the first time that I work with CF so I don't know what the best practices are and the best way to do things, I'll do some research and come back with a better approach